ranger/data/scope.sh

```diff
@@ -93,23 +117,13 @@ handle_extension() {
             ## Preview as text conversion (unsupported by pandoc for markdown)
             odt2txt "${FILE_PATH}" && exit 5
             exit 1;;
-
         ## XLSX
         xlsx)
             ## Preview as csv conversion
             ## Uses: https://github.com/dilshod/xlsx2csv
             xlsx2csv -- "${FILE_PATH}" && exit 5
             exit 1;;
 
-        ## HTML
-        htm|html|xhtml)
```

Comment:
Here we remove support of html command-line browsers & replace it with a preview of html file, not html page.
I think this is wrong, and we should leave this part as is, maybe add optional setting to view html as text, not as page

Reply:
Again, you are proposing removing a case of use that affect 99% of users, to ship `html command-line browser support`, which is a 1% of users case of use. And which people interested easily enable anyway editing their `scope.sh`.